jgrapht-core/src/main/java/org/jgrapht/generate/WindmillGraphsGenerator.java

```diff
@@ -40,8 +40,12 @@
 public class WindmillGraphsGenerator<V, E>
     implements GraphGenerator<V, E, V>
 {
-
-    enum Mode
+    /**
+     * WINDMILL and DUTCHWINDMILL Modes for the Constructor
+     * @author Philipp
```

Comment:
we usually don't add an @author in an inner enum.

Reply:
removed.